portfolio/src/main/webapp/index.html

```diff
@@ -9,7 +9,7 @@
 
 
 
-  <body bgcolor ="AliceBlue">
+  <body  class ="serif" bgcolor ="AliceBlue">
```

Comment:
Nitpicking: try to keep the single-space-between-attributes styling consistent. That means remove the extra space between 'class'.

Reply:
Quick question: what does "serif" stand for?

Reply:
Serif is a font family. I was applying it using a function within my style.css file.

Reply:
I see. Thanks.